app/main.py

```diff
@@ -1,4 +1,8 @@
 def get_human_age(cat_age: int, dog_age: int) -> list:
+    if not isinstance(dog_age, int) or not isinstance(cat_age, int):
+        raise ValueError
+    if cat_age < 0 or dog_age < 0:
+        raise ValueError
```

Comment:
Similarly, include a message with this `ValueError` to indicate why the error is being raised (e.g., 'Age cannot be negative').